src/AboutPage/About.tsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function About() {
   return (
-    <div>About</div>
+    <body>
+      <section className='flex justify-center items-center h-screen w-screen flex-col'>
+        <h1 className=' text-5xl m-10'>About</h1>
+        <img src="/src/images/hero-img.png" className='w-5/6 h 3/6'></img>
+      </section>
+
+      <section className='flex flex-col justify-center items-center'>
+        <article className='flex flex-row w-5/6 m-20'>
```

Comment:
There should be a gap set here. Note that if you resize the screen and the text/image get close together, there's no space between them.